Source/Xamarin/Prism.Ninject.Forms/PrismIocExtensions.cs

```diff
@@ -5,12 +5,12 @@ namespace Prism.Ninject
 {
     public static class PrismIocExtensions
     {
-        public static IKernel GetContainer(this IContainerProvider containerProvider)
+        public static IKernel GetKernel(this IContainerProvider containerProvider)
```

Comment:
Not sure why you renamed this?  Now this API is not consistent with the rest.  This should be called `GetContainer`. Remember, this is an abstraction and not meant to represent the actual container being used.